docs/admin/kube-proxy.md

```diff
@@ -11,8 +11,8 @@ notitle: true
 
 The Kubernetes network proxy runs on each node. This
 reflects services as defined in the Kubernetes API on each node and can do simple
-TCP, UDP stream forwarding or round robin TCP, UDP forwarding across a set of backends.
-Service cluster IPs and ports are currently found through Docker-links-compatible
+TCP, UDP stream forwarding or round robin TCP,UDP forwarding across a set of backends.
+Service cluster ips and ports are currently found through Docker-links-compatible
```

Comment:
Should we use lower case `ip` here?

Reply:
All of this content is from generated source - like the whitespace request, relevant changes will need to be in kubernetes/kubernetes, but I'm not entirely sure where. I'll research this further and see where changes need to be made.

Reply:
@xiangpengzhao https://github.com/kubernetes/kubernetes/pull/53671 has been opened to solve upstream issue here